source/initial_composition/ascii_data.cc

```diff
@@ -59,6 +76,31 @@ namespace aspect
         Utilities::AsciiDataBase<dim>::declare_parameters(prm,
                                                           "$ASPECT_SOURCE_DIR/data/initial-composition/ascii-data/test/",
                                                           "box_2d.txt");
+
+        prm.enter_subsection("Ascii data model");
+        {
+          prm.declare_entry("Slice dataset in 2D plane", "false",
+                            Patterns::Bool (),
+                            "Whether to use a 2D data slice from a 3D data file "
+                            "or the entire data file. Slicing a 3D dataset is "
+                            "only supported for 2D models.");
+          prm.declare_entry ("First point on slice", "0.0,1.0,0.0",
```

Comment:
Can you add a comment that these parameters will be ignored if "Slice dataset in 2D plane" is false.